lib/MetaCPAN/Server/Model/CPAN.pm

```diff
@@ -11,9 +11,9 @@ extends 'Catalyst::Model';
 with 'CatalystX::Component::Traits';
 
 has esx_model => (
-    is         => 'ro',
-    lazy_build => 1,
-    handles    => ['es'],
+    is      => 'ro',
+    handles => ['es'],
+    default => sub { MetaCPAN::Model->new( es => $_[0]->servers ) },
```

Comment:
I already reverted (1a7a689) and fixed this one (259f6ef), as it broke the web server startup